index.html

```diff
@@ -4040,7 +4040,7 @@ <h3 id="methods-1">Methods</h3>
                                         <p>Use this object's <var>cdm instance</var> to process <var>sanitized certificate</var>.</p>
                                     </li>
                                     <li>
-                                        <p>If the preceding step failed, resolve <var>promise</var> with a new <code><a href="https://heycam.github.io/webidl/#dfn-DOMException">DOMException</a></code> whose name is the appropriate <a href="#error-names">error name</a>.</p>
+                                        <p>If the preceding step failed, reject <var>promise</var> with a new <code><a href="https://heycam.github.io/webidl/#dfn-DOMException">DOMException</a></code> whose name is the appropriate <a href="#error-names">error name</a>.</p>
```

Comment:
This file is supposed to be generated (see https://github.com/w3c/encrypted-media/blob/master/TEAM.md). In this case, since this is a post-REC change, it might be nice to keep it small like this. However, we should update the date.